src/storage/BaseProcessor.inl

```diff
@@ -146,5 +146,116 @@ void BaseProcessor<RESP>::doRemoveRange(GraphSpaceID spaceId,
     });
 }
 
+
+template<typename RESP>
+StatusOr<std::pair<std::string, std::string>> BaseProcessor<RESP>::assembleIndex(
```

Comment:
the caller do not need the error info, maybe can return std::pair<std::string, std::string> directly, and empty mean assembleIndex failed.

Reply:
Actually, I disagree.

Reply:
Because I check all the caller, all of them never check the return info, and do not log the error info, so I think it is useless, maybe we can log it, then if there are some problems, we can fix it by the log.